master/buildbot/test/__init__.py

```diff
@@ -119,3 +119,6 @@
 
 # FIXME: needs to be sorted out (#3666)
 warnings.filterwarnings('ignore', "Invalid utf8 character string")
+
+# twisted.compat.execfile is using 'U' https://twistedmatrix.com/trac/ticket/9023
```

Comment:
This is triggering lint errors.  Not sure what you want to do here, but I say to waste less time,
leave it out, and for now only test Py3 against Twisted from trunk.